client/my-sites/domains/domain-management/list/item.jsx

```diff
@@ -159,29 +159,30 @@ class ListItem extends React.PureComponent {
 
 		if ( domain.expired ) {
 			return (
-				<Notice isCompact status="is-error" icon="spam">
-					{ translate( 'Expired %(timeSinceExpiry)s', {
+				<DomainNotice
+					status="alert"
+					text={ translate( 'Expired %(timeSinceExpiry)s', {
```

Comment:
:warning: This change will be queued for retranslation. We'll display the English text until then.